runtime/src/iree/tooling/vm_util.cc

```diff
@@ -46,7 +46,6 @@ static iree_status_t CreateBufferViewFromFile(
         IREE_STATUS_RESOURCE_EXHAUSTED,
         "a shape rank of %zu is just a little bit excessive, eh?", shape_rank);
   }
-  shape_result = iree_status_ignore(shape_result);
```

Comment:
this is required - the if above is only guarding against a failure that is not out-of-range; if it is out of range then we need to free the status

Reply:
Re-added the iree_status_ignore but without the assignment.